include/FreeRTOSIPConfigDefaults.h

```diff
@@ -524,6 +524,22 @@
     #define ipconfigIP_TASK_STACK_SIZE_WORDS    ( configMINIMAL_STACK_SIZE * 5 )
 #endif
 
+/* Default the size of the stack used by the EMAC deferred handler task to twice
+ * the size of the stack used by the idle task - but allow this to be overridden in
+ * FreeRTOSIPConfig.h as ipconfigEMAC_TASK_STACK_SIZE_WORDS is a user definable constant.
+ * Note that the macro 'configEMAC_TASK_STACK_SIZE' was used in earlier versions.
+ * When defined, it will be used, albeit with a #warning.
+ */
+#ifndef ipconfigEMAC_TASK_STACK_SIZE_WORDS
+    /* Check if the earlier name of the macro is still defined. */
+    #ifdef configEMAC_TASK_STACK_SIZE
+        #warning This macro is now called ipconfigEMAC_TASK_STACK_SIZE_WORDS
```

Comment:
I am not sure that this would work with visual studio.
Do we need to add a pragma message instead? Something like this: https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/b6eac0ca7df8c8935cb840fd9642f6533a0da2e6/tools/tcp_utilities/tcp_mem_stats.c#L61?